src/intl/en/page-run-a-node.json

```
@@ -0,0 +1,115 @@
+  "page-run-a-node-rasp-pi-note-2-description": "Learn how to set up a node via the command line on a Raspberry Pi",
+  "page-run-a-node-rasp-pi-note-3-link": "Run a node with Raspberry Pi",
+  "page-run-a-node-rasp-pi-note-3-description": "Follow along here if tutorials are your thing",
+  "page-run-a-node-shop": "Shop",
+  "page-run-a-node-shop-avado": "Shop Avado",
+  "page-run-a-node-shop-dappnode": "Shop DAppNode",
+  "page-run-a-node-staking-title": "Stake your ETH",
+  "page-run-a-node-staking-description": "Though not required, with a node up and running you're one step closer to staking your ETH to earn rewards and help secure Ethereum.",
+  "page-run-a-node-staking-link": "Stake ETH",
+  "page-run-a-node-staking-plans-title": "Plan on staking?",
+  "page-run-a-node-staking-plans-description": "To maximize the efficiency of your validator, a minimum of 16GB RAM is recommended, but 32GB is better, with a CPU benchmark score of 6667+ on cpubenchmark.net. It is also recommended that stakers have access to unlimited high-speed internet bandwidth, though this is not an absolute requirement.",
+  "page-run-a-node-sovereignty-title": "Sovereignty",
+  "page-run-a-node-sovereignty-preview": "Think of running a node like the next step beyond getting your own Ethereum wallet.",
+  "page-run-a-node-sovereignty-1": "An Ethereum wallet allows you to take full custody and control of your digital assets by holding the private keys to your addresses, but those keys don't tell you the current state of the blockchain, such as your wallet balance.",
+  "page-run-a-node-sovereignty-2": "By default, Ethereum wallets typically reach out to a 3rd-party node, such as Infura or Alchemy, when looking up your balances. Running your own node allows you to have your own copy of the Ethereum blockchain.",
+  "page-run-a-node-title": "Run a node",
+  "page-run-a-node-voice-your-choice-title": "Voice your choice",
+  "page-run-a-node-voice-your-choice-preview": "Don't give up control in the event of a fork.",
+  "page-run-a-node-voice-your-choice-1": "In the event of a chain fork, where two chains emerge with two different sets of rules, running your own node guarantees your ability to choose which set of rules you support. It’s up to you to upgrade to new rules and support proposed changes, or not.",
+  "page-run-a-node-voice-your-choice-2": "If you're staking ETH, running your own node allows you to chose your own client, to minimize your risk of slashing and to react to fluctuating demands of the network over time. Staking with a third party forfeits your vote on which client you think is the best choice.",
+  "page-run-a-node-what-title": "<i>What</i> does it mean to \"run a node\"?",
+  "page-run-a-node-what-1-subtitle": "Run software.",
+  "page-run-a-node-what-1-text": "Known as a \"client,\" this software downloads a copy of the Ethereum blockchain, keeps it up-to-date with new blocks and transactions, and helps others download and update their own copies.",
```

> downloads a copy of the Ethereum blockchain, keeps it up-to-date with new blocks and transactions '

One important fact I would add here is that client (cryptographically, mathematically) verifies this data.

---

Thanks for the feedback @taxmeifyoucan .. Yeah, can't disagree too strongly. Been a little back and forth on this statement, trying to keep it really simple for people in this section. Will keep this in mind for sure as we get other feedback. 👍🏻

---

I agree - came here to say that. Maybe something like:

> Known as a "client," this software downloads a copy of the Ethereum blockchain, keeps it up to date by verifying new activity, and helps others download and update their own secure copies.

---

On the other hand, I'm pretty fine with what you have. I'm guessing these users will understand what transactions & blocks are...

---

Just tossing out another take

> Known as a \"client,\" this software downloads a copy of the Ethereum blockchain and verifies the validity of every block, then keeps it up-to-date with new blocks and transactions, and helps others download and update their own copies.
```suggestion
  "page-run-a-node-what-1-text": "Known as a \"client,\" this software downloads a copy of the Ethereum blockchain and verifies the validity of every block, then keeps it up-to-date with new blocks and transactions, and helps others download and update their own copies.",
```

---

LGTM @wackerow 👍